src/index.ts

```diff
@@ -300,6 +300,13 @@ export type {
   SupportedTLSSocketOptions,
   WithSessionCallback
 } from './mongo_client';
+export type {
+  MongoLoggableComponent,
+  MongoLogger,
+  MongoLoggerMongoClientOptions,
+  MongoLoggerOptions,
+  SeverityLevel
+} from './mongo_logger';
```

Comment:
im exporting the necessary types here for API extractor because if logging is enabled, the `mongoLogger` prop will be of type `MongoLogger`. without this API extractor errors. is there something else i should be doing instead?

Reply:
This is related to the api-extractor issues we discussed last week at tech chat.  I'm consider this resolved but am leaving it open as context for other reviewers.